json/tbd_seb_n4.json

```diff
@@ -0,0 +1,67 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "testing TBD JSON for seb.osm",
+  "psis": [
+    {
+      "id": "bad",
+      "fenestration": 1.1
+    },
+    {
+      "id": "good",
+      "parapet": 0.5,
+      "rimjoist": 0.9
+    },
+    {
+      "id": "compliant",
+      "rimjoist": 0.3,
+      "parapet": 0.325,
+      "fenestration": 0.35,
+      "concave": 0.45,
+      "convex": 0.45,
+      "balcony": 0.5,
+      "party": 0.5,
+      "grade": 0.45
+    }
+  ],
+  "khis": [
+    {
+      "id": "column",
+      "point": 0.0
+    },
+    {
+      "id": "support",
+      "point": 0.0
+    }
+  ],
+  "edges": [
+    {
+      "psi": "bad",
+      "type": "fenestration",
+      "surfaces": [
+        "Entryway  Wall 5",
+        "Sub Surface 8"
```

Comment:
The approach of specifying edges by two surfaces (as opposed to an edge id) is growing on me, I like it because you can find that information from just the OSM/IDF.  Just to make sure, this `psi` would apply to all edges shared between these two surfaces in the case that they share more than one edge right?

Reply:
You got it. The surface list is not a bad approach in the future for third-party GUIs (e.g. plugin), or even manual editing of the JSON file, e.g. a user can easily figure out linked surface names (e.g. OpenStudio plugin inspector, interacting with the geometry model under the OpenStudio Applications Geometry tab). To your 2nd question, correct - all edges matching the criteria (in the example above, all 4x :fenestration perimeter edges) will use the custom edge :psi and/or :type. To narrow down the targeted edge list, one can specify the edge length (e.g. "3.55", which would rule out the 2x shorter window jambs).